helm/reana/README.md

```diff
@@ -16,7 +16,7 @@ This Helm automatically prefixes all names using the release name to avoid colli
 | `components.reana_server.environment.REANA_MAX_CONCURRENT_BATCH_WORKFLOWS` | Upper limit on concurrent REANA batch workflows running in the cluster. | 30 |
 | `components.reana_server.environment.REANA_USER_EMAIL_CONFIRMATION` | Enable user to confirm their email address. | true |
 | `components.reana_server.environment.REANA_SCHEDULER_REQUEUE_SLEEP` | Seconds to wait between consuming workflows. | 15 |
-| `components.reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT` | The number of times to requeue workflow before failing it. | 200 |
+| `components.reana_server.environment.REANA_SCHEDULER_REQUEUE_COUNT` | The number of times to requeue workflow before failing it. You can use value "infinity" to never fail. | 200 |
```

Comment:
Minor suggestion:
`"infinity" value could be used to deactivate workflow failing.`